app/views/home/about.html.haml

```diff
@@ -14,8 +14,22 @@
     semua orang. Selain itu, kontes ini berisi soal-soal berkualitas
     yang sangat cocok untuk persiapan OSN tingkat kabupaten, provinsi,
     maupun nasional.
+
   %h2 Tim Kami
-  .row#daftar-panitia
-  \... dan banyak lainnya!
-  %h3 Alumni Tim Kami
-  .row#alumni-panitia
+  - team = @about_users.select { |data| !data.is_alumni }
+  - team.each_with_index do |data, index|
+    - base_class = "about-us-person col-md-4 col-sm-8"
+    %div{:class => "#{base_class}", :data => {:name => "#{data.name}", :description => "#{data.description}"}}
```

Comment:
You can do something like `%div.#{base_class}` or something I think. Try to look into other HAML files.

Reply:
I think the ones with extra html tags (such as `:data` and `:description`) need to be done in this way, not sure if a better way such as `%div.#{base_class}` exist for those.

Reply:
I see. One way is to abstract this thing into a helper. Anyway leaving it like this is also fine, but I think it's better for the class to be put as `.#{base_class}` because it's rather special compared to the data attributes.